src/snabbdom.ts

```diff
@@ -297,6 +297,10 @@ export function init (modules: Array<Partial<Module>>, domApi?: DOMAPI) {
       oldVnode = emptyNodeAt(oldVnode);
     }
 
+    if (!(vnode && isVnode(vnode))) {
+      vnode = emptyNode;
+    }
```

Comment:
I would check specifically for `vnode === null`.

Reply:
@mightyiam doesn't it instead make sense to add the null check inside of `isVnode` ?

Reply:
That doesn't feel right to me. It's not whether it's a VNode. It's whether it's specifically `null`.